packages/databricks-vscode/src/logger/LoggerManager.ts

```diff
@@ -25,40 +32,49 @@ export class LoggerManager {
         return logFile;
     }
 
-    private getFileTransport(
-        filename: string,
-        extraOptions?: Omit<
-            transports.FileTransportOptions,
-            "filename" | "format"
-        >
+    private getLogOutputChannel(
+        name: "Databricks Logs" | "Databricks Bundle Logs"
     ) {
-        return new transports.File({
-            format: format.combine(format.timestamp(), format.json()),
-            filename: filename,
-            ...extraOptions,
-        });
+        if (!this.outputChannels.has(name)) {
+            const outputChannel = window.createOutputChannel(name, {log: true});
+            outputChannel.clear();
+            this.outputChannels.set(name, outputChannel);
+        }
+        return this.outputChannels.get(name)!;
     }
 
     async initLoggers() {
-        const outputChannel = window.createOutputChannel("Databricks Logs");
-        outputChannel.clear();
-
-        const sdkAndExtensionLogfile =
-            await this.getLogFile("sdk-and-extension");
+        const commonLogTransports = [
+            ...LOG_OUTPUT_CHANNEL_LEVELS.map(
+                (level) =>
```

Comment:
Why do we create different transports for each level? Might be good to add a comment with an explanation here

Reply:
We need this because the output console requires different functions to print different levels of log. Since it is called from the stream, which does not get the log level as input ever, we need to specify the log level of the stream at object initialisation time.